packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.service.js

```diff
@@ -53,4 +54,80 @@ export default class SofpthoneService {
       .get(`/telephony/${billingAccount}/line/${serviceName}/devices`)
       .then(({ data }) => data);
   }
+
+  getThemes() {
+    return this.$http.get('/telephony/softphone/themes').then(({ data }) => {
+      return Promise.all(
```

Comment:
```suggestion
      return this.$q.all(
```

Reply:
Silly question but what is the advantage of $q over Promise ?